app/uk/gov/hmrc/tai/forms/EditIncomeForm.scala

```diff
@@ -103,12 +104,13 @@ object EditIncomeForm {
       payToDate)
   }
 
-  def bind(taxablePayYTD: BigDecimal = BigDecimal(0), payDate: Option[LocalDate] = None,
-           errMessage: Option[String] = None)(implicit request: Request[_], messages: Messages) = createForm(taxablePayYTD, payDate, errMessage).bindFromRequest
+  def bind(employerName: String, taxablePayYTD: BigDecimal = BigDecimal(0), payDate: Option[LocalDate] = None,
+           errMessage: Option[String] = None)(implicit request: Request[_], messages: Messages) = createForm(employerName, taxablePayYTD, payDate, errMessage).bindFromRequest
 
-  private def createForm (taxablePayYTD: BigDecimal, payDate: Option[LocalDate] = None, errMessage: Option[String] = None)(implicit messages: Messages) :Form[EditIncomeForm] = {
+  private def createForm (employerName: String, taxablePayYTD: BigDecimal, payDate: Option[LocalDate] = None, errMessage: Option[String] = None)(implicit messages: Messages) :Form[EditIncomeForm] = {
 
     val monthName = payDate.map (date => DateHelper.monthOfYear(Dates.formatDate(date))).getOrElse("")
+    val yearName = payDate.map (date => DateHelper.getYear(Dates.formatDate(date))).getOrElse("")
```

Comment:
Maybe could combine these into one method in the date helper monthAndYear which removes the responsibility from here.